io/driver/module_unload_load.sh.data/README.txt

```diff
@@ -0,0 +1,3 @@
+This Program loads and unloads the kernel driver modules. For the kernel driver modules that have dependencies, the dependant modules should  be listed in the config file (config). The script takescare of unloading the dependant modules, if any, before unloading the core module. It runs for a total of 100 times as stress.
+
```

Comment:
You should also mention, that this test passes when at least one of the drivers finishes the whole loop.

Additionally it'd be worth mentioning that it picks only modules listed in `lspci`.

Apart from those little clarifications I think it's ready for inclusion.